ecspresso run fails without --propagate-tags flag

# Version

v1.99.0

# Summary

I understand that if `--propagate-tags` is not specified, the propagateTags api parameter is set to NONE by default, but for some reason, without the flag, the request fails with an invalid value error.
If I specify `--propagate-tags TASK_DEFINITION`, it succeeds.

# Error output example

```console
$ ecspresso run --config ecspresso.yml
2022/11/09 17:51:25.531482 ecspresso-test/api Running task
2022/11/09 17:51:25.534606 ecspresso-test/api Registering a new task definition...
2022/11/09 17:51:26.460098 ecspresso-test/api Task definition is registered ecspresso-test:48
2022/11/09 17:51:26.679482 ecspresso-test/api Task definition ARN: arn:aws:ecs:us-east-1:xxxx:task-definition/ecspresso-test:48
2022/11/09 17:51:26.679549 ecspresso-test/api Watch container: hello
2022/11/09 17:51:26.679844 ecspresso-test/api Running task with arn:aws:ecs:us-east-1:xxxx:task-definition/ecspresso-test:48
2022/11/09 17:51:26.931123 [ERROR] run FAILED. failed to run task: operation error ECS: RunTask, https response error StatusCode: 400, RequestID: ddb230d3-e92d-4656-877b-1ffa3f58cade, InvalidParameterException: Invalid value for propagateTags
```

Thanks!
But when I tried v1.99.1 , got a panic by SIGSEGV.

```console
$ ecspresso run --config ecspresso.yml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x20f70e9]

goroutine 1 [running]:
github.com/kayac/ecspresso.(*App).Start(0x0?, {0x2769338?, 0xc000525380?})
        github.com/kayac/ecspresso/ecspresso.go:131 +0x29
github.com/kayac/ecspresso.(*App).Run(0x0?, {0x2769338?, 0xc000525380?}, {0xc000235761, 0xc00009dc40, 0xc000235771, 0xc00009dc90, 0xc00009dcc0, 0xc000235789, 0xc0002357a0, ...})
        github.com/kayac/ecspresso/run.go:43 +0x66
github.com/kayac/ecspresso.dispatchCLI({0x2769338, 0xc000525380}, {0xc0001f9246, 0x3}, 0xc0001de1c0)
        github.com/kayac/ecspresso/cli.go:114 +0x27c
github.com/kayac/ecspresso.CLI({0x2769338, 0xc000525380}, 0x315bd00?)
        github.com/kayac/ecspresso/cli.go:150 +0x96
main.main()
        github.com/kayac/ecspresso/cmd/ecspresso/main.go:25 +0xce
```

So, I tried v1.99.0 with only the following changes applied and built it, and run succeeded.

```diff
-	case "":
-		in.PropagateTags = types.PropagateTagsNone
+	case "", "NONE":
+		// XXX ECS says > InvalidParameterException: Invalid value for propagateTags
+		// in.PropagateTags = types.PropagateTagsNone
+		in.PropagateTags = ""
```

Another commit or change seems to be the cause of the panic.

--- a/config.go
+++ b/config.go
@@ -125,6 +125,12 @@ func (c *Config) Restrict(ctx context.Context) error {
 		}
 		c.versionConstraints = constraints
 	}
+	if c.Timeout == nil {
+		c.Timeout = &Duration{Duration: DefaultTimeout}
+	}
+	if c.Region == "" {
+		c.Region = os.Getenv("AWS_REGION")
+	}
 	var err error
 	c.awsv2Config, err = awsConfig.LoadDefaultConfig(ctx, awsConfig.WithRegion(c.Region))
 	if err != nil {

--- a/config_test.go
+++ b/config_test.go
@@ -246,3 +246,27 @@ func TestConfigWithInvalidRequiredVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadConfigWithoutTimeout(t *testing.T) {
+	region := os.Getenv("AWS_REGION")
+	os.Setenv("AWS_REGION", "ap-northeast-2")
+	defer func() { os.Setenv("AWS_REGION", region) }()
+
+	ctx := context.Background()
+	loader := ecspresso.NewConfigLoader(nil, nil)
+	conf, err := loader.Load(ctx, "tests/notimeout.yml", "")
+	if err != nil {
+		t.Log("unexpected an error", err)
+		t.FailNow()
+	}
+	if conf.Timeout == nil {
+		t.Error("expected default timeout, but nil")
+	}
+	if conf.Timeout.Duration != ecspresso.DefaultTimeout {
+		t.Errorf("expected default timeout, but %v", conf.Timeout.Duration)
+	}
+
+	if conf.Region != "ap-northeast-2" {
+		t.Errorf("expected region from AWS_REGION, but %v", conf.Region)
+	}
+}

--- a/orb.yml
+++ b/orb.yml
@@ -18,7 +18,7 @@ commands:
           command: |
             VERSION="<< parameters.version >>"
             if [ "${VERSION}" = "latest" ]; then
-              DOWNLOAD_URL=$(curl -sS https://api.github.com/repos/kayac/ecspresso/releases/latest|jq -r '.assets[].browser_download_url|select(match("linux.amd64."))')
+              DOWNLOAD_URL=$(curl -sS https://api.github.com/repos/kayac/ecspresso/releases | jq -r '[.[]|select(.tag_name > "v1.99")][0].assets[].browser_download_url|select(match("linux.amd64."))')
             else
               DOWNLOAD_URL=$(curl -sS https://api.github.com/repos/kayac/ecspresso/releases/tags/${VERSION}|jq -r '.assets[].browser_download_url|select(match("linux.amd64."))')
             fi

--- a/tests/notimeout.yml
+++ b/tests/notimeout.yml
@@ -1,0 +1,4 @@
+cluster: default
+service: test
+service_definition: ecs-service-def.json
+task_definition: ecs-task-def.json

